### PR TITLE
RavenDB-15791 Certificates View: Enhancements

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/manage/certificates.ts [hunk 1]
```diff
@@ -217,8 +235,11 @@ class certificates extends viewModelBase {
     }
     
     enterEditCertificateMode(itemToEdit: unifiedCertificateDefinition) {
-        this.model(certificateModel.fromDto(itemToEdit));
-        this.model().validationGroup.errors.showAllMessages(false);
+        if (!_.includes(itemToEdit.Thumbprints, this.serverCertificateThumbprint())) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: do we need this check here? I think we should just disable button?

File: src/Raven.Studio/typescript/viewmodels/manage/certificates.ts [hunk 2]
```diff
@@ -391,8 +413,17 @@ class certificates extends viewModelBase {
                 });
                 
                 mergedCertificates = _.sortBy(mergedCertificates, x => x.Name.toLocaleLowerCase());
-                this.updateCache(mergedCertificates);
-                this.certificates(mergedCertificates);
+                
+                // place the server certificate and client certificate first in list
+                let serverCert = mergedCertificates.find(x => _.includes(x.Thumbprints, this.serverCertificateThumbprint()));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: use const?

**Review by ml054 — APPROVED** [review]

File: src/Raven.Studio/typescript/viewmodels/shell.ts [hunk 3]
```diff
@@ -258,25 +259,24 @@ class shell extends viewModelBase {
 
         if (this.clientCertificate() && this.clientCertificate().Name) {
             
-            const dbAccess = certificateModel.resolveDatabasesAccess(this.clientCertificate())
-                .map(x => `<div>${x}</div>`)
-                .join("");
-            
+            const dbAccess = certificateModel.resolveDatabasesAccess(this.clientCertificate());
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/shell.ts:262]: update this method to never return null.

File: src/Raven.Studio/typescript/viewmodels/manage/certificates.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: removed

File: src/Raven.Studio/typescript/viewmodels/manage/certificates.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/manage/certificates.ts]: done

File: src/Raven.Studio/typescript/viewmodels/shell.ts (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/shell.ts:262]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**